core/tests/test_common.py

```diff
@@ -0,0 +1,16 @@
+import geopandas
+import momepy
+
+import core
+
+
+class TestContinuity:
+    def setup_method(self):
+        self.city = "Liège"
+        self.roads = momepy.remove_false_nodes(core.utils.read_parquet_roads(self.city))
+
+    def test_basic(self):
+        roads = core.algorithms.common.continuity(self.roads)
+
+        assert isinstance(roads, geopandas.GeoDataFrame)
+        assert roads.geom_type.unique()[0] == "LineString"
```

Comment:
Can this also test that new columns have been added? I would leave the tests of correctness to momepy, maybe with an exception of `coins_end` which is defined here.

Reply:
done in [d0eab9c](https://github.com/uscuni/simplification/pull/80/commits/d0eab9c806cefa51fe0f9c10f97c67baf52d0a8d)